tox.ini

```diff
@@ -0,0 +1,12 @@
+[tox]
+envlist =
+    py{27,33,34,35,36,37}
+
+[testenv]
+setenv =
+    PYTHONDONTWRITEBYTECODE=1
```

Comment:
I cant see a strong reason to keep it, I guess its better to remove it and find out for myself what the benefit is